src/commands/math/basicSymbols.js

```diff
@@ -526,3 +528,26 @@ LatexCmds['÷'] = LatexCmds.div = LatexCmds.divide = LatexCmds.divides =
   bind(BinaryOperator,'\\div ','&divide;', '[/]');
 
 CharCmds['~'] = LatexCmds.sim = bind(BinaryOperator, '\\sim ', '~', '~');
+
+for (var cmd in LatexCmds) {
+  if (LatexCmds.hasOwnProperty(cmd)) {
+      constructor = LatexCmds[cmd];
+      var block = constructor();
```

Comment:
this variable seems like it should be called command or symbol or something. Block in MathQuill specifically means a MathBlock object containing zero or more commands, whereas this is exactly one command object